code/index.html

```diff
@@ -4,9 +4,64 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link type="text/css" rel="stylesheet" href="style.css">
     <title>Project 4: Pizzabot</title>
   </head>
   <body>
-    <h1>This is your Pizza Bot</h1>
+    <section class="main-container">
+        <img src="./img/pizza1.jpg" alt="image of pizza">
+    <!--container for the form-->
+    <div class="form-container">
+      <div class="content-container">
+        <div class="content-text">
+          <h1>Pizza Time!</h1>
+          <p id="welcomeMessage"></p> 
+        </div>
+    <!--form that gets user input, id:s connects with javascript and assigns value to variables.-->
+        <form id="form-action">      
+          <fieldset>
+            <legend>Order here</legend>
+            <div class="form-action-group">
+              <div class="choosePizza"> 
+                <p id="choosePizzaText"></p>
+              </div>
+            
+              <label for="pizzaSort"></label>
+                <input 
+                type="text"
+                name="pizzaSort"
+                id="pizzaSort"
+                placeholder="e.g. Hawaiian Pizza"
+                />
```

Comment:
Happy to see proper code formatting, but all attributes should be two spaces more to the right than their opening tag